plugins/services/src/js/components/modals/ServiceModals.js

```diff
@@ -232,6 +247,7 @@ class ServiceModals extends React.Component {
         {this.getResumeModal()}
         {this.getScaleModal()}
         {this.getStopModal()}
+        {this.getResetDelayAction()}
```

Comment:
I realize that this is weird, but as far as I can tell, the action dropdown is made to work with modals and we don't have a modal here, so we are just executing an action. If two reviewers are against this approach, I will try again to make this work in some other way (and help in that will be appreciated).